ui/app/templates/vault/cluster/metrics/index.hbs

```diff
@@ -21,18 +21,20 @@
           Vault usage
         {{/link-to}}
       {{/link-to}}
-      {{#link-to
-        "vault.cluster.metrics.config"
-        tagName="li"
-        activeClass="is-active"
-      }}
+      {{#if model.config.configPath.canRead}}
         {{#link-to
```

Comment:
link-to within another link-to, is that right, or am I reading it wrong?

Reply:
This was copypasta but you're right, it doesn't look like it needs to have both nested 🤔

Reply:
Just confirmed, this is how the tabs are in `secret-list-header` component, and removing the inner one causes strange styling issues. Gonna leave it for now, but worth looking into in the future